cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -108,3 +108,11 @@ keystore_pass: changeme
 # like CentOS or RHEL <= 7, this is the yum conf file, but for Fedora and the
 # like, this is the dnf conf file
 version_pin_file: /etc/yum.conf
+# This is the port Jenkins will listen on for slaves.
+jenkins_slave_agent_port: 50000
+# This is a list of ports to allow in firewalld.
+firewall_ports:
```

Comment:
Alternatively, we can change this from just a list of port numbers to a list of port/protocol numbers.